validator/java/.gitignore

```diff
@@ -0,0 +1,6 @@
+# These directories get generated by Maven.
+target/
+src/main/resources/
+src/main/java/dev/amp/validator/ValidatorProtos.java
+bazel-*
+bin
```

Comment:
Add these directories to `gulp clean`. (Another reason in favor of consolidating into one directory at a later point, if possible.)

https://github.com/ampproject/amphtml/blob/06b6ed4d1e59df8697994b5473d223fae9504c14/build-system/tasks/clean.js#L20-L37

Reply:
Done!